### 获取A股股票列表

In [1]:
import akshare as ak
all_stocks = ak.stock_zh_a_spot_em()[['代码', '名称']]
print(all_stocks.head(5))

  0%|          | 0/57 [00:00<?, ?it/s]

       代码    名称
0  920045  N蘅东光
1  301687  N新广益
2  920270  天铭科技
3  920130  立方控股
4  300058  蓝色光标


### A股股票to csv

结果示例：

代码,名称

920045,N蘅东光

301687,N新广益

920270,天铭科技

In [2]:
import pandas as pd

all_stocks = ak.stock_zh_a_spot_em()[['代码', '名称']]
all_stocks.to_csv('data/all_a_stocks.csv', index=False)

  0%|          | 0/57 [00:00<?, ?it/s]

### read_csv 功能学习
要特别注意pd.read_csv 会把000123这种读成123，所以要定义为字符串